### ISSUE TITLE
8$: improve diagnostic for ',' instead of ';' in interface body

### ISSUE THREAD

**Issue by strager** [quick-lint/quick-lint-js#990]
```typescript
interface ChartSeriesProps<Sample> extends NodeProps {
  getX: (sample: Sample) => number;
  getY: (sample: Sample) => number;
  samples: Sample[],
}
```

actual:
> chart/src/scenes/example.tsx|128 col 20 error| E0209: commas are not allowed between class methods
> chart/src/scenes/example.tsx|128 col 20-19 error| E0223: missing semicolon after field

expected:
> chart/src/scenes/example.tsx|128 col 20 error| E0209: commas are not allowed between interface members

**Comment by JPMoles** [comment]
So it's producing diagnostic E0223 when it shouldn't?

**Comment by strager** [comment]
> So it's producing diagnostic E0223 when it shouldn't?

Correct.

The minimal fix for the example TypeScript code is to remove the comma, not to add a semicolon.
